### PR TITLE
[ActionSheet] Disable a iOS13 failing test.

### PR DESCRIPTION
**Pull request by fluxminds**
Part of `testDefaultBackgroundColor` is failing because of XCode beta. Disable this test to make iOS13 bot happy.

Error description:
Setting `UIColor.white` as backgroundColor on a tableView causes `<UIDynamicSystemColor: 0x7fab88c49d50; name = tableBackgroundColor>` to be set in the end. It seems to be an XCode 11 beta issue.

### CODE REVIEW

**Review by codeman7 — APPROVED** [review]

File: components/ActionSheet/tests/unit/ActionSheetTest.swift [hunk 1]
```diff
@@ -90,10 +90,13 @@ class ActionSheetTest: XCTestCase {
     // Then
     XCTAssertEqual(actionSheet.backgroundColor, .white)
     XCTAssertEqual(actionSheet.view.backgroundColor, .white)
-    let subviewsArray = actionSheet.view.subviews
-    for view in subviewsArray {
-      XCTAssertEqual(view.backgroundColor, .white)
-    }
+
+    //  Disable this test because of XCode beta.
+    //  Tracking issue (https://github.com/material-components/material-components-ios/issues/8238)
```
Comment (jverkoey) [inline components/ActionSheet/tests/unit/ActionSheetTest.swift]: Please mark this using TODO notation:

```
// TODO(https://github.com/material-components/material-components-ios/issues/8238): Re-enable this test.
```

**Comment by jverkoey** [comment]
Please include the error that this PR is addressing in the PR description.

File: components/ActionSheet/tests/unit/ActionSheetTest.swift (hunk 1 above)
Comment (fluxminds) [reply components/ActionSheet/tests/unit/ActionSheetTest.swift]: Addressed in 72283648bacf7efcd28cbd4b0af638582f37ab03

**Comment by fluxminds** [comment]
PR description has been updated.

**Pull request merged**